NEWS

```diff
@@ -53,7 +53,8 @@ NEWS - user visible changes				-*- outline -*-
    These formats come with Area A enforcement in the PROCEDURE
    DIVISION, where paragraph and section names should start in Area A
    (i.e, before margin B).  The underlying checks can be enabled or
-   disabled using a new flag `areacheck`.
+   disabled using a new flag `areacheck`, or with `>>AREACHECK` and
```

Comment:
The reference format is already a dialect option, and `-f(no-)areacheck` has an effect only based on the format. Are there dialects with format not in {free,fixed}, and for which area A enforcement should not be enabled by default?

Reply:
Hum I see: you'd want the `areacheck` dialect option to be about support for `>>AREACHECK` directives and the likes? This may prevent one from being able to completely disable such checks without manually adding the directives, for instance on `cobol85` source, right? (or is this what `-fareacheck=ignore` would be about?).

Reply:
> Hum I see: you'd want the areacheck dialect option to be about support for `>>AREACHECK` directives and the likes? 

No!

Dialect comes first, then command line options, then directives (`$SET [NO]AREACHECK` being ignored when there is no area as you've implemented is, is of course correct).

The point of a dialect option would be to have it default to "on" for some of dialects (ibm and mvs, possibly gcos?).
As you've noted a boolean dialect option would be reasonable.